Libplanet/Blockchain/BlockChain.cs

```diff
@@ -69,30 +74,40 @@ public class BlockChain<T>
         /// it checks if the existing genesis block and this argument is the same.
         /// If the <paramref name="store"/> has no genesis block yet this argument will
         /// be used for that.</param>
+        /// <param name="allowRender">Defines whether to allow rendering to be performed on this
+        /// <see cref="BlockChain{T}"/>.</param>
         /// <exception cref="InvalidGenesisBlockException">Thrown when the <paramref name="store"/>
         /// has a genesis block and it does not match to what the network expects
         /// (i.e., <paramref name="genesisBlock"/>).</exception>
         public BlockChain(
             IBlockPolicy<T> policy,
             IStore store,
-            Block<T> genesisBlock
+            Block<T> genesisBlock,
+            bool allowRender = true
```

Comment:
How about renaming it to `render` which is simpler?

Reply:
Renamed the parameter to `render`.